tests/test_pdf_download.py

```diff
@@ -0,0 +1,269 @@
+from .data.pdf_download_data import HTML_TEXT, YEAR_LINKS
+
+class TestCheckPageSetup:
+    """Tests check_page_setup() which confirms that the current layout of
+    the BOE page still matches the expected layout
+    """
+
+    def test_boe_link(self):
+        """Tests that the link supplied directs to BOE meetings page of the
+        Comptroller website
+
+        TEST DATA
+        - N/A
+
+        TEST SETUP
+        - N/A
+
+        ASSERTIONS
+        - assert that the response from hitting the page is 200
+        - assert that the page contains "Minutes" in the page title
+        """
+        assert 1
+
+    def test_year_div(self):
+        """Tests that the div containing the links to the minutes page for
+        each year exists on the page
+
+        TEST DATA
+        - N/A
+
+        TEST DATA
+        - N/A
+
+        ASSERTIONS
+        - assert that a div with class 'field field-name-body'
+        - assert that there is a <p> tag with links that follow a year format
+        """
+        assert 1
+
+    def test_minute_div(self):
+        """Tests that the div containing the links to each meeting's minutes
+        exists on the page
+
+        TEST DATA
+        - N/A
+
+        TEST DATA
+        - N/A
+
+        ASSERTIONS
+        - assert that a div with class 'field field-name-body'
+        - assert that there is a <p> tag with links that follow the
+          meeting link format
+        """
+        assert 1
+
+class TestGetYearLinks:
```

Comment:
One question that I have is whether the names of these classes have a one-to-one relationship with certain functions in the code base, and whether the naming schemes are related. Let's make sure to define this questions and add to workflow documentation.